gensim/scripts/glove2word2vec.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Radim Rehurek <radimrehurek@seznam.cz>
+# Copyright (C) 2016 Manas Ranjan Kar <manasrkar91@gmail.com>
```

Comment:
Just checking that you are aware of the legal guidelines of contributing. "By submitting your contribution to be included in the gensim project, you agree to assign joint ownership of your changes (your code patch, documentation fix, whatever) to me, Radim Řehůřek."

https://github.com/piskvorky/gensim/wiki/Developer-page#legal